benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -9,6 +9,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseEventID": "createTestCase", "UserRole": "caseworker-sscs", "CRUD": "R"},
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "postponementRequest", "UserRole": "caseworker-sscs-dwpresponsewriter", "CRUD": "CRUD"},
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "postponementRequest", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequest", "UserRole": "caseworker-sscs-judge", "CRUD": "CRUD"},
```

Comment:
I don't see any mentions on the ticket about this

Reply:
[SSCS-10789](https://tools.hmcts.net/jira/browse/SSCS-10789)

Reply:
Ideally one PR should address one ticket, but let's get this one out of the way. I'll mark this conversation as resolved.